src/Common/src/Interop/Unix/System.Net.Security.Native/NegotiationInfoClass.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    // This class is used to determine if NTLM or
+    // Kerberos are used in the context of a Negotiate handshake
+    internal class NegotiationInfoClass
```

Comment:
For Windows, it is under Interop\Windows\SspiCli. Any recommendations for Unix? (it is needed for both SPNEGO and NTLM protocols)